docs/actions-docker.md

```diff
@@ -236,6 +236,18 @@ DATE=`date`
 echo "{ \"message\": \"Hello $NAME! It is $DATE.\" }"
 ```
 
+For the return result, not only support `dictionary` but also support `array`
+```
+#!/bin/bash
+echo '["a", "b"]''
+```
+
+And support array result for sequence action as well, the first action's array result can be used as next action's input parameter
+```
+#!/bin/bash
+echo $1
```

Comment:
This is sequence action's second action. can accept the array as input parameter, and we can do some sort operation here,
But here, i just return it directly(don't add sort operation),  just a example.

Reply:
But there seems to be a bad point here, it implies that any type of parameter can be received, but my example's exact meaning is, it here just can accept array type.